### DISCUSSION TITLE
Add prompt reload after each batch iteration option to use batch with wildcards

### DISCUSSION THREAD

**Discussion by Anonym121** [Mikubill/sd-webui-controlnet#1662] [Ideas]
Hello, for a while, I have been struggling to increase the quantity of good images I get in txt2img.
I noticed that if I use openpose, canny or other controlnet in batch using a folder with my good images, I can increase the overall quality of the new ones and I see a decrease of bad legs, arms and other bad features.
However, I extensively use wildcards to add variance and get the AI to combine ideas humans would never expect and once batch controlnet starts, it chooses a value for each wildcard and a seed, which makes homogeneous outputs.
I noticed there is a check box to increase the seed with each batch controlnet iteration. 
So my request is to add a checkbox or script that reloads prompt after each controlnet iteration.

This would allow making a library of poses or compositions and using it with wildcards to add different characters to the image and/or with different clothing for example.

Kindly thank you for taking your time to read this.

**Comment by ghost** [comment]
If you use dynamic prompting for wildcards you can tick the box "Unlink seed from prompt" in the extension fields and it should generate always random prompts. Had the same problem and it wasnt possible to do with Unprompted extension

That being said I was able to code a "choose random pic" for Controlnet (chooses random file from specified batch folder) function and its really great. But the one-line code solution caused some bugs so I had to revert it. Lets have a "pick random from batch" option!
